### ISSUE TITLE
single object tracking

### ISSUE THREAD

**Issue by feixiangdekaka** [xingyizhou/CenterTrack#10]
If the centermask  can  use for single  object teacking.

**Comment by rlrahulkanojia** [comment]
yes, there is a hyperparameter K in opts.py which describes the number of outputs.
but which object to choose may be a tricky part.

**Comment by xingyizhou** [comment]
I haven't tried. A simple implementation is still tracking all objects, but just pick one as output...

**Comment by dongziqi001** [comment]
@xingyizhou  just pick one as output
can you tell me which part to change in the code??
